nucypher/blockchain/eth/actors.py

```diff
@@ -218,7 +218,7 @@ def __get_deployer(self, contract_name: str):
         return Deployer
 
     @staticmethod
-    def __collect_deployment_secret(deployer) -> str:
+    def collect_deployment_secret(deployer) -> str:
```

Comment:
No more privacy?

Reply:
Yeah, I needed to use this function outside of the" automated series of deployments" (`deploy_network_contracts`) method below this one in the CLI when deploying an individual upgradeable contract, and collecting it's secret.